ios/brave-ios/swiftlint.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/sh
-
-#
-# Runs SwiftLint to enforce style guides
-#
-# Adds support for Apple Silicon brew directory
-export PATH="$PATH:/opt/homebrew/bin"
-
-if which swiftlint; then
```

Comment:
Is swiftlint preserved or we stop using it?

Reply:
For now we will remove it, Chromium iOS only relies on `swiftformat` for their Swift code which I will be enabling after this migration completes (so that it doesnt run on every Swift file we have now).

I may bring back swiftlint in the future and add it as a custom PRESUBMIT check though. It requires pulling the tool through DEPS/runhooks though so I left it for now